common/views/themes/typography.ts

```diff
@@ -242,6 +242,17 @@ export const typography = css<GlobalStyleProps>`
       ${fontSizeMixin(2)}
     }
 
+    .content-type-visual-story & h2 {
```

Comment:
TODO: This could probably do with a comment to explain that it's a variant of content typography styles